main.go

```diff
@@ -44,50 +44,48 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(machinev1.AddToScheme(scheme))
-	// +kubebuilder:scaffold:scheme
+	//+kubebuilder:scaffold:scheme
 }
 
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
```

Comment:
If this argument changes name, then the `Deployment` definition which supplies that argument will also need to be updated to the new name:
https://github.com/openshift/managed-node-metadata-operator/blob/c8fcec810e57c236b1fad02ef804c53ccd47846d/deploy/20_operator.yaml#L35

Reply:
Thank you @mrbarge for the catch. Applied fix in my latest commit